Makefile

```diff
@@ -1,10 +1,19 @@
 RUNC_TEST_IMAGE=runc_test
 PROJECT=github.com/opencontainers/runc
 TEST_DOCKERFILE=script/test_Dockerfile
+VERSION := $(shell cat VERSION)
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
+GITCOMMIT := $(shell git rev-parse --short HEAD)
+GITUNTRACKEDCHANGES := $(shell git status --porcelain --untracked-files=no)
+ifneq ($(GITUNTRACKEDCHANGES),)
+	GITCOMMIT := $(GITCOMMIT)-dirty
+endif
```

Comment:
Any way we could make these `git` parts work even if we don't have `.git` available? (ie, a snapshot downloaded from GitHub's archives)